Haskell/syntax_test_haskell.hs

```diff
@@ -93,21 +102,28 @@
 --
    a a = (+) a 2
 --     ^ keyword.operator.haskell
```

Comment:
Shouldn't this be `keyword.operator.assignment.haskell`? Also perhaps `keyword.operator.declaration.haskell`.